ims/einstein/operations.py

```diff
@@ -217,11 +217,16 @@ def shutdown(self):
 
     # Provisions from HaaS and Boots the given node with given image
     @log
-    def provision(self, node_name, img_name, network, nic):
+    def provision(self, node_name, img_name, network, nic, hil_flag):
         try:
+            hil_bool = bool(hil_flag == "True")
```

Comment:
I am guessing in picasso/rest.py. In the _rest_wrapper if we wrap the args with json.loads, then probably that should fix it.